docs/source/modin/supported/series_str_supported.rst

```diff
@@ -44,7 +44,8 @@ the method in the left column.
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``fullmatch``               | N                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``get``                     | N                               |                                                    |
+| ``get``                     | P                               | ``N`` if the `i` parameter is set to a non-int     |
```

Comment:
Current implementation also doesn't support lists and dict. Shall we mention that here?

Reply:
Good point. Updated the docs.